src/BondingCurveRegistry.sol

```diff
@@ -21,14 +20,11 @@ import {Errors} from "src/libraries/Errors.sol";
  *         You can think of the registry as a concierge the EthMultiVault uses to access various
  *         economic incentive patterns.
  */
-contract BondingCurveRegistry is Initializable {
+contract BondingCurveRegistry {
```

Comment:
3.18 - Missing Inheritance From Interface

It looks like we elected not to make the BondingCurveRegistry inherit from the IBondingCurveRegistry interface.  @mihailo-maksa Can you confirm if this should be here?  It may have gotten obliterated by an auto-merge.